Packages/com.unity.inputsystem/InputSystem/Actions/InputActionAsset.cs

```diff
@@ -83,6 +83,8 @@ public class InputActionAsset : ScriptableObject, IInputActionCollection2
         /// InputActionAssets.
         /// </remarks>
         public const string Extension = "inputactions";
+        ////REVIEW: actually pre-populate with some stuff?
```

Comment:
What is the content after a save of an empty asset? Not sure why do not initialise it like that when its created? Then we wouldn't need these checks at all and the equality check would solve that for us

Reply:
Check https://github.com/Unity-Technologies/InputSystem/pull/1761#discussion_r1345711499